index.js

```diff
@@ -52,21 +56,21 @@ function checkStatus (bot, message, whatElse, more) {
         nolimit.stash({ key: message.comment.user, value: 1 })
         sendMessage(bot, randomWinningPhrase(message))
       } else {
-        sendMessage(bot, 'Sorry <@' + message.comment.user + '>, you cant dibs-a-bot until the boss says so, and right now he says no.')
+        sendMessage(bot, `Sorry <@${message.comment.user}>, you cant dibs-a-bot until the boss says so, and right now he says no.`)
       }
     }
   } else {
-    sendMessage(bot, 'Sorry <@' + message.comment.user + '>, this item is claimed.')
+    sendMessage(bot, `Sorry <@${message.comment.user}>, this item is claimed.`)
   }
 }
 
 function randomWinningPhrase (message) {
   var phrases = [
-    'Break out the BLUE LABEL!!! <@' + message.comment.user + '> is the winner.',
-    'Congrats <@' + message.comment.user + '>, ' + admin1 + ' doesn’t want something anymore and you\'re closer than a trash can.',
-    'Hey Hey Hey <@' + message.comment.user + '>! Enjoy the trash that someone else didn\'t want.',
-    'Well aren\'t you lucky <@' + message.comment.user + '>! You\'ve won at thing of semi-value.',
-    'Whoa <@' + message.comment.user + '>!, thank you for slightly reducing the Fire Hazard around ' + admin1
+    `Break out the BLUE LABEL!!! <@${message.comment.user}> is the winner.`,
+    `Congrats <@${message.comment.user}>, <@${params.admin.primary}> doesn’t want something anymore and you\'re closer than a trash can.`,
+    `Hey Hey Hey <@${message.comment.user}>! Enjoy the trash that someone else didn\'t want.`,
+    `Well aren\t you lucky <@${message.comment.user}>! You\'ve won at thing of semi-value.`,
+    `Whoa <@${message.comment.user}>!, thank you for slightly reducing the Fire Hazard around <@${params.admin.primary}>`
   ]
```

Comment:
@jescalan you say the ${} way is the newer standard correct? i'll run with that if so